affinity/cpp-20/d0796r2.md

```diff
@@ -533,23 +461,10 @@ With the ability to place memory with affinity comes the ability to define algor
 |------------|
 | Should the interface provide standard algorithms or policies for distributing memory? |
 
-## Level of abstraction
-
-The current proposal provides an interface for querying whether an `execution_resource` can allocate and/or execute work, it can provide the concurrency it supports and it can provide a name. We also provide the `affinity_query` structure for querying the relative affinity metrics between two `execution_resource`s. However this may not be enough information for users to take full advance of the system, they may also want to know what kind of memory is available or the properties by which work is executed. It was decided that attempting to enumerate the various hardware components would not be ideal as that would make it harder for implementers to support new hardware. It has been discussed that a better approach would be to parameterise the additional properties of hardware such that hardware queries could be much more generic.
+## Dynamic topology discovery - deferred
```

Comment:
I think we should leave in the discussion of dynamic topology discovery, as it's an important topic to SG1, we want to show it's still on the roadmap but we just haven't gotten to it yet.

Reply:
Still in the intro material.  Seemed prudent to mention but not spend any SG1 time discussing.